### ISSUE TITLE
Using Unicode block characters would result in higher resolution graphs - improvement

### ISSUE THREAD

**Issue by rbanffy** [bminixhofer/performance#2]
We could use the quadrants (U+2596-U+259F) for higher value (2x) and temporal (2x) resolutions. Using eights (U+2581-U+2588) would yield 8x higher value resolution but no time resolution improvement.

The Terminals Working Group also has a proposal (not accepted just yet - http://www.unicode.org/L2/L2017/17435-terminals-prop.pdf) that would include Teletext ITU T.100 mosaic characters (3x higher value, 2x higher time) resolutions, the T.101 smoothed mosaic characters (same resolution but prettier), as well as improved eights.

The graphs would need to be filled for the current eights and the proposed smoother mosaic chars, but not for the current quadrants and proposed sextants.

**Comment by bminixhofer** [comment]
Thanks Ricardo, good suggestion. I was unsure about which unicode characters to use, great that someone knowing their stuff chimed in :)

I would tend towards switching to quadrants for now and later to sextants or mosaics once the proposal is accepted. 

In my opinion for performance monitoring a balance between time and value resolution is good as none is clearly more important than the other so we should prefer quadrants over eights (but both are clearly better than the current solution).

Whats your opinion on this? And would you have time to implement it?

**Comment by rbanffy** [comment]
I agree that quadrants, then T.100 mosaics, would be a good idea. The proposal also includes PETSCII and ATASCII chars that also have their uses, so I think the best approach would be to have command-line switches handle the selection, defaulting to quadrants (which are already Unicode compliant and should be considered the baseline) and enable others as they become available (and font support comes up).
